bin/update-ip.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Gets the ip address of the server container and sets it in place in
+# your /etc/hosts file.
+
+IPBASE=`/sbin/ifconfig docker0 | grep 'inet addr:'| awk '{print $2}' | sed -e 's/.*://g' | cut -d '.' -f 1-2`
+NEWIP=`docker inspect ideatown_server_1 | grep '"IPAddress' | awk '{sub(/["|,]/, "", $2); print $2}' | sed 's/["|,]//g'`
```

Comment:
http://linuxconfig.org/how-to-retrieve-docker-container-s-internal-ip-address

I think you may be able to get the IPAddress using a custom formatter:
```sh
$ docker inspect -f '{{ .NetworkSettings.IPAddress }}' ideatown_server_1
172.17.0.2
```